dbld/bootstrap

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+cd /source
+./autogen.sh
+
+cd /build
+rm -rf *
```

Comment:
Just mark this rm -rf for the same rethinking as in the previous patch.

Reply:
ok